draft-ietf-ppm-dap.md

```diff
@@ -110,11 +110,11 @@ revealing any individual user's data.
 
 This document describes the Distributed Aggregation Protocol (DAP) for privacy
 preserving measurement. The protocol is executed by a large set of clients and a
-small set of servers. The servers' goal is to compute some aggregate statistic
-over the clients' inputs without learning the inputs themselves. This is made
-possible by distributing the computation among the servers in such a way that,
-as long as at least one of them executes the protocol honestly, no input is ever
-seen in the clear by any server.
+small set of aggregator servers. The aggregators' goal is to compute some
```

Comment:
(This comment follows from off-PR discussion with/suggestion by @jbr.)

"small set of aggregator servers" is correct but perhaps imprecise -- nowadays, there are exactly two aggregator servers, the Leader & the Helper. Maybe "two aggregator servers, called the Leader & the Helper"?

Reply:
Makes sense! I left out the discussion of leader and helper because those terms aren't used again in the introduction and they get introduced properly in the glossary.